jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -122,7 +125,8 @@ public void testPluginConfigurationProcessor_defaults()
       throws InvalidImageReferenceException, IOException, CacheDirectoryCreationException,
           MainClassInferenceException, InvalidAppRootException, InvalidWorkingDirectoryException,
           InvalidContainerVolumeException, IncompatibleBaseImageJavaVersionException,
-          NumberFormatException, InvalidContainerizingModeException {
+          NumberFormatException, InvalidContainerizingModeException,
+          InvalidFilesModificationTimeException {
```

Comment:
I wonder if we should have a `JibConfigurationException` base class.

Reply:
Yeah, it is about time to do something about handling invalid config values. I'll create an issue as a tech debt. Hopefully we get to it soon.